src/elements/module.rs

```diff
@@ -324,6 +324,22 @@ impl Module {
 		None
 	}
 
+	/// True if a name section is present.
+	///
+	/// NOTE: this can return true even if the section was not parsed, hence `names_section()` may return `None`
+	///       even if this returns `true`
+	pub fn has_names_section(&self) -> bool {
+		self.sections().iter().position(|e| {
```

Comment:
nit: Can `position + is_some` be replaced with `any`?

Reply:
Changed! Also added tests.